package.json

```diff
@@ -2,26 +2,28 @@
     "name": "apify-cli",
     "version": "0.19.3",
     "description": "Apify command-line interface (CLI) helps you manage the Apify cloud platform and develop, build, and deploy Apify Actors.",
-    "main": "index.js",
+    "exports": "./dist/index.js",
+    "types": "dist/index.d.ts",
+    "type": "module",
     "scripts": {
         "test": "cross-env APIFY_CLI_SKIP_UPDATE_CHECK=1 vitest run",
         "test-python": "cross-env APIFY_CLI_SKIP_UPDATE_CHECK=1 vitest run -t '.*\\[python\\]'",
         "lint": "eslint src test",
         "lint:fix": "eslint src test --fix",
-        "commands-md": "npm run manifest && oclif-dev readme",
         "prepare-release": "npm run commands-md && npm run prod-shrinkwrap",
-        "prod-shrinkwrap": "rm -rf node_modules && npm install --production && npm shrinkwrap",
-        "manifest": "rm -f oclif.manifest.json && oclif-dev manifest",
-        "postinstall": "node \"./src/bin/run\" check-version && node ./src/lib/community"
+        "clean": "rimraf dist",
+        "build": "npm run clean && tsc",
+        "postpack": "rimraf oclif.manifest.json",
+        "prepack": "npm run build && oclif manifest && oclif readme",
+        "prepare": "npm run build && node ./bin/run.js check-version && node ./dist/lib/community.js"
     },
     "files": [
-        "src",
+        "dist",
         "oclif.manifest.json",
-        "index.js",
-        "package.json"
+        "bin"
     ],
     "bin": {
-        "apify": "./src/bin/run"
+        "apify": "./bin/run.js"
```

Comment:
I just applied what they said to do through their migration guides, as well as the example repo (https://github.com/oclif/hello-world-esm/tree/main/bin)
I'll try this version on windows and will check

Reply:
I just know that it didn't work this way, the windows binary wasn't picked automatically. I can imagine people don't care about this, because you usually don't need the windows binary. But if you have that `-S` in the hashbang (as we have in the `dev.js` script), that is problematic and you need the windows script (and then you can fall into those problems). I'd expect the regular `run.js` script to work fine even if you remove the `run.esm`.

Reply:
The dev script is honestly kinddd of useless, only we run it locally, users should never encounter or use it. Users that run the cli won't have any arguments in the hashbang, as that runs node directly with the built code, but I'll try it out tonight and see what happens

Reply:
yeah, i know all this only because the mikro orm cli supports TS out of box, and the regular npm bin script is using the same hashbang as the dev script here (and it failed on windows).